### ISSUE TITLE
Mod stopped to work on current version

### ISSUE THREAD

**Issue by paulocampoy** [CorsairCat/DCS-A-6E-Intruder#2]
Try to run this mod using DCS version 2.7.8.16140 and DCS is freezing.
Any tip?

**Comment by CorsairCat** [comment]
should be worked now

**Comment by paulocampoy** [comment]
ok... will test... was working until 2.7.5.10869

**Comment by CorsairCat** [comment]
Yep, it should be the lods issues, which happens on our WIP SK60 too. Just check if the freeze was happens when the loading page is finished?

**Comment by paulocampoy** [comment]
Perfect the freeze is fixed...  I've tested on version 2.7.8.16140.
I attached some video on links with "new bugs" I believe correlated with this shapes fix.
Now the problem are:
- a black square on game screen where is the HUD; ( Link for video [https:/we.tl/t-ahRgWiOgDL](https:/we.tl/t-ahRgWiOgDL) )
- if select outside plane view, the plane dissapear; ( Link for video [https:/we.tl/t-ahRgWiOgDL](https:/we.tl/t-ahRgWiOgDL) )
- on a mission that I created to take off from a Carrier (with no SuperCarrier DLC) the plane drop and pass through the carrier deck and drop on water. ( Link for video [https:/we.tl/t-9AZiMuHSeB](https:/we.tl/t-9AZiMuHSeB) )

I attached too the videos of old version of shapes that was working until version 2.7.5 and I attached too the files with missions that I used on tests.

- files used on all tests ( Link for files [https://we.tl/t-SkslFbmLmT](https://we.tl/t-SkslFbmLmT) )
- on 2.7.5 with no fix shape file for Free Flight using file ( Link for video [https:/we.tl/t-9AZiMuHSeB](https:/we.tl/t-9AZiMuHSeB) )
- on 2.7.5 with no fix shape file for Carrier take off using file ( Link for video [https:/we.tl/t-jt55KYVHzU](https:/we.tl/t-jt55KYVHzU) )

Best Regards,
